tests/x11/firefox_audio.pm

```diff
@@ -18,16 +18,18 @@ use testapi;
 
 sub run {
     my ($self) = @_;
-    $self->start_firefox();
-    send_key "ctrl-l";
-    type_string(autoinst_url . "/data/1d5d9dD.oga");
-    send_key "ret";
     start_audiocapture;
+    x11_start_program("firefox " . autoinst_url . "/data/1d5d9dD.oga");
+    $self->firefox_check_default;
+    $self->firefox_check_popups;
     assert_screen 'test-firefox_audio-1', 35;
     sleep 1;    # at least a second of silence
     assert_recorded_sound('DTMF-159D');
     send_key "alt-f4";
-    $self->exit_firefox();
```

Comment:
Why not to keep it? exit_firefox method does exactly what we need, including alt-f4 key presses

Reply:
This is a bit of a tardy answer, as the PR is now closed, but the main reason I removed the exit_firefox() method here is that it expects an open xterm needle after closing firefox. That works well when complementing the start_firefox() method, but not so well when launching ffox from the desktop launcher.

Therefore if I had used the exit_firefox() method, it would have needed adjusting, which I deemed overkill and inconsistent, as there was already an argument against using the generic method in the first place - so I simply opted for reverting the previous [change](https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/6401f7049b53c070cff4c1614cb87fe5ace16faa#diff-33406501a27b0aa9d7631e5a8037b2c4).